### PR TITLE
add missing constants

### PR DESCRIPTION
**Pull request by nduchastel**
Adding new constants missing.
Was causing mypy errors since it did not know E_SSL_CONNECT_ERROR.
Based on comments from gregprice, I added all of them.
(BTW, found a small typo in 'INFILESIE_LARGE' (aka needs to be 'INFILESIZE_LARGE')

### CODE REVIEW

File: third_party/2/pycurl.pyi [hunk 1]
```diff
@@ -45,37 +45,339 @@ class CurlShare(object):
...
+FTP_CREATE_MISSING_DIRS = ... # type: int
+FTP_ENTRY_PATH = ... # type: int
+FTP_FILEMETHOD = ... # type: int
+FTP_RESPONSE_TIMEOUT = ... # type: int
+FTP_SKIP_PASV_IP = ... # type: int
+FTP_SSL = ... # type: int
+FTP_SSL_CCC = ... # type: int
+FTP_USE_EPRT = ... # type: int
+FTP_USE_EPSV = ... # type: int
+GLOBAL_ALL = ... # type: int
```
Comment (ambv) [inline third_party/2/pycurl.pyi]: This is redefined from line 7. The tests complain about a few more. Make sure to ./runtests.sh locally.

**Comment by nduchastel** [comment]
Added the extra space.
Remove duplicates GLOBAL_...

**Comment by nduchastel** [comment]
it's been a while since I did pull requests... I think it is ready for acceptance and merging into master branch. Let me know if I need to do anything else.

**Comment by ambv** [comment]
Indeed. Happy holidays and thank you for your contribution!

**Pull request merged**